src/main/models/tasks.ts

```diff
@@ -0,0 +1,136 @@
+/* eslint-disable no-use-before-define */
+import {
+  DataTypes,
+  InferAttributes,
+  InferCreationAttributes,
+  Model,
+  Sequelize,
+} from 'sequelize';
+import { SchemaAPIRoute, TaskSchema } from 'src/shared/generated_models';
+import MLModelDb from './ml-model';
+
+/*
+(taskUid,modelUid) - primary key
+shortTitle
+taskRoute
+taskOrder
+taskSchema
+*/
+
+class TaskDb extends Model<
+  InferAttributes<TaskDb>,
+  InferCreationAttributes<TaskDb>
+> {
+  declare uid: string;
+
+  declare modelUid: string;
+
+  declare shortTitle: string;
+
+  declare taskRoute: string;
+
+  declare taskOrder: number;
+
+  declare taskSchema: TaskSchema; // JSON String
+
+  public static getAllTasks() {
+    return TaskDb.findAll();
+  }
+
+  public static async getTaskByUid(uid: string) {
+    return TaskDb.findOne({
+      where: {
+        uid,
+      },
+    });
+  }
+
+  public static getTaskByModelUid(modelUid: string) {
+    return TaskDb.findAll({
+      where: {
+        modelUid,
+      },
+    });
+  }
+
+  public static createTask(
+    modelUid: string,
+    schemaApiRoute: SchemaAPIRoute,
+    taskSchema: TaskSchema,
+  ) {
+    const {
+      short_title: shortTitle,
+      run_task: taskRoute,
+      order: taskOrder,
+    } = schemaApiRoute;
+
+    const uid = String(taskOrder);
+
+    return TaskDb.create({
+      uid,
+      modelUid,
+      shortTitle,
+      taskRoute,
+      taskOrder,
+      taskSchema,
+    });
+  }
+
+  public static deleteTask(uid: string) {
+    return TaskDb.destroy({
+      where: {
+        uid,
+      },
+    });
+  }
+}
+
+export const initJob = async (connection: Sequelize) => {
```

Comment:
This should be `initTasks` (or `initTask`).

Reply:
Thanks for the catch!